paper/tsd2017.tex

```diff
@@ -235,14 +239,15 @@ \subsection{Unified Simplified Grapheme Modeling}\label{usg}
 \end{enumerate}
 %It is motivated by the intuition that ...
 For that we used a joint acoustic model of Czech, Hungarian and Polish.
+% szerintem ezt az alábbi 2 részt egybe kellene venni. Hiszen a két lépés inkább egyszerre, vagy legalábbis fordított sorrendben történik. Amúgy nem egymáshoz hasonlítjuk a nyelvek grafémáit, hanem a latinhoz és ami eltér, azt egyszerűsítjük.
```

Comment:
Értettem, messziről jól is nézett ki, de a mögöttes tartalommal nem jön össze. Itt szakiknak magyarázunk, a túlzott szájbarágás nem jó (és ha logikai bukfenc van, akkor pláne).